database/lorem_options.js

```diff
@@ -0,0 +1,40 @@
+let randomInt = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```

Comment:
if you wanted to  dry your code here as well (since you have randomInt function in another file) could make all of these options something that you generate with a function that you call from that file with the randomInt there. It looks like you either have all the options of just the first two, so I suppose you could generate an object with the only the parameters passed in. Or not - this is very readable as it is.